hushline/templates/settings/branding.html

```diff
@@ -40,6 +40,18 @@ <h3>App Name</h3>
     <button type="submit" class="btn">Update Name</button>
   </form>
 
+  <h3>Directory Introduction Text</h3>
+  <form
+    method="POST"
+    action="{{ url_for('settings.update_directory_intro_text') }}"
+    class="formBody"
+  >
+    {{ update_directory_intro_text_form.hidden_tag() }}
+    <label for="directory-intro-text">Introduction Text</label>
```

Comment:
this can be achieved with `{{ update_directory_intro_text_form.directory_intro_text.label }}`

Reply:
7bd531d